src/styles/components/_card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  width: 20rem;
+  margin: auto;
+  padding: 3.2rem 1.6rem 1.6rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  @include border(0.1rem, $color-card-bgc, 0.5rem);
+
+  &__image {
+    width: 16rem;
+    height: 13.4rem;
+    background-image: url('../images/imac.jpeg');
+    background-size: cover;
+    margin-bottom: 4rem;
+
+    @extend %bgc-image;
+  }
+
+  &__name,
+  &__reviews,
+  &__price.value {
+    color: $color-txt-dark;
+  }
+
+  &__reviews,
+  &__prices {
+    @extend %flex-between-center;
+  }
+
+  &__code,
```

Comment:
The class `&__prices` is mentioned here, but there is no corresponding style or use of this class elsewhere in this file. Please ensure that you are using the correct class name and that it is necessary.